python-sdk/examples/strands_bot.py

```diff
@@ -5,24 +5,19 @@
 
 from openinference.instrumentation.litellm import LiteLLMInstrumentor
 
-from typing import cast
 from dotenv import load_dotenv
 
 load_dotenv()
 
 import chainlit as cl
-import litellm
-from litellm import CustomStreamWrapper
-from litellm.types.utils import StreamingChoices
-from strands.telemetry.tracer import get_tracer
-
-tracer = get_tracer(
-    service_name="strands-bot",
-    otlp_endpoint=f"{os.environ.get('LANGWATCH_ENDPOINT', 'https://app.langwatch.ai')}/api/otel/v1/traces",
-    otlp_headers={"Authorization": "Bearer " + os.environ["LANGWATCH_API_KEY"]},
-    enable_console_export=True,  # Helpful for development
+from strands.telemetry import StrandsTelemetry
+
+strands_telemetry = StrandsTelemetry().setup_otlp_exporter(
+    endpoint="https://app.langwatch.ai/api/otel/v1/traces",
+    headers={"Authorization": f"Bearer {os.environ['LANGWATCH_API_KEY']}"},
 )
 
+
 class KiteAgent:
     def __init__(self):
         langwatch.setup(instrumentors=[LiteLLMInstrumentor()])
```

Comment:
@rogeriochaves when I use this instrumentor I get the following error in all my traces:
RecursionError: maximum recursion depth exceeded while encoding a JSON object

Without it, while metadata isn't being attached, the agent runs fine.